src/edges/edges.js

```diff
@@ -22,27 +22,29 @@ const getInterests = (interests) => {
 
 const getStroke = (shareValues) => {
   const { exact, minimum, maximum } = shareValues || {};
-  if (exact === undefined) {
+  if (exact === undefined && minimum <= maximum) {
     if (minimum !== undefined && maximum !== undefined) {
       return (minimum + maximum) / 2 / 10;
     } else {
       return DEFAULT_STROKE;
     }
   } else {
```

Comment:
Suggest adding another if clause here to deal with scenario in which minimum > maximum